vet.sh

```diff
@@ -65,7 +65,7 @@ git ls-files "*.go" | xargs sed -i 's:"golang.org/x/net/context":"context":'
 set +o pipefail
 # TODO: Stop filtering pb.go files once golang/protobuf#214 is fixed.
 # TODO: Remove clientconn exception once go1.6 support is removed.
```

Comment:
looks like you addressed this TODO.

Reply:
The comment was removed when Go1.6 support was removed.